src/platform.ts

```diff
@@ -130,13 +130,19 @@ export class HomebridgeMagichomeDynamicPlatform implements DynamicPlatformPlugin
           newDevices++;
           
         } else {
-          if(existingAccessory.context.device.deviceData == undefined) {
```

Comment:
deviceData was never added to the object, so the repair was triggering at every start